superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -107,6 +105,13 @@ const StyledAnchor = styled.a`
 
 const { SubMenu } = Menu;
 
+interface ExtentionConfigs {
+  CSV_EXTENSIONS: Array<any>;
```

Comment:
these should be `boolean`

Reply:
No these are for what are coming back from the backend configs which are arrays of extension types.